src/core/find_HII_bubbles.c

```diff
@@ -65,7 +65,7 @@ void _find_HII_bubbles(const int snapshot)
   double density_over_mean;
   double weighted_sfr_density;
   double f_coll_stars;
-  double neutral_fraction;
+  double electron_fraction;
```

Comment:
this is neutral_fraction. am i misunderstanding something?

Reply:
I think it is neutral fraction but for some reason in Balu's version was called electron fraction so I changed the name for consistency... Do you want me to put it back?

Reply:
mmmmm are you sure they are the same? If we are also taking into account He, they electron fraction is slightly more than the neutral fraction. Maybe this was named so for future development as well?

Reply:
https://github.com/s-balu/_meraxes_/blob/98d1d31b7ccda8bdf4230361dea86cced48ca57c/src/core/find_HII_bubbles.c#L412

It is set from the x_e value...

Reply:
@s-balu i think you misunderstood electron_fraction. electron_fraction is the fraction of electron, we are considering neutral hydrogen fraction. x_e is also the electron density within HI region.

Reply:
Well in that case it makes sense to revert (?) back to neutral_fraction.

Reply:
Ok so I will revert to neutral fraction and then happy to merge